### PR TITLE
make prompt icons more easily configurable

### PR DESCRIPTION
**Pull request by coryb**
@AlecAivazis, here is the follow up on #45 to make all the icons more easily customizable by users.

### CODE REVIEW

File: core/template.go [hunk 1]
```diff
@@ -11,7 +11,16 @@ var DisableColor = false
...
-	HelpIcon      = "ⓘ"
+
+	ErrorIcon    = "✘"
+	HelpIcon     = "ⓘ"
+	QuestionIcon = "?"
+
+	MarkedOptionIcon   = "◉"
+	UnmarkedOptionIcon = "◯"
+
+	MultiSelectFocusIcon = "❯"
```
Comment (AlecAivazis) [inline core/template.go]: Should these be the same for consistency?

Comment (coryb) [inline core/template.go]: Yeah, I was thinking that, consistency would probably be a good idea.  Which one wins?

Comment (AlecAivazis) [inline core/template.go]: imo `MultiSelectFocusIcon` is a pretty clear winner

Comment (coryb) [inline core/template.go]: Cool, my preference also, PR incoming :)

**Pull request merged**